indico/modules/events/abstracts/operations.py

```diff
@@ -54,6 +54,17 @@ def create_abstract(event, abstract_data, custom_fields_data=None):
     return abstract
 
 
+def update_abstract(abstract, data):
+    abstract.person_links = []
```

Comment:
still need to check if there's a better way to avoid this workaround. it's the old problem with sqlalchemy where replacing a relationship's data may result in the new data being inserted before the old has been removed. or maybe we  somehow accidentally create new abstract person links instead of reusing the existing ones...